perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -90,6 +95,27 @@ def AptInstall(vm):
   _Install(vm)
 
 
+def JujuInstall(vm):
+    if vm.isController is True:
```

Comment:
This only gets called on the controller because the cassandra_stress benchmark calls 

vm_util.RunThreaded(lambda vm: vm.Install('cassandra'), benchmark_spec.vms)

In general you can't expect benchmarks to install all packages on all VMs. Instead, calling install on a  (non-controller) juju vm should probably trigger the controller vm installing the package on all vms in that group - it would be nice if this were generalized so that benchmarks don't have to explicitly support it